test/specs/oboe.component.spec.js

```diff
@@ -1,3 +1,31 @@
+import {
```

Comment:
I'm seeing an error coming from this file when I run `npm run test`
```
ERROR in ./test/specs/oboe.component.spec.js
Module not found: Error: Can't resolve '../../dist/oboe-node.js' in '/Users/juan/code/personal/oboe.js/test/specs'
```

The code between lines 29 and 74 was a hacked together from when I was trying to upgrade jasmine. I think with this import work, it can probably be scrapped all together and we'll just rely on the imports

Reply:
I taken that code out now and it is relying only on the imports. That error does not happen not.

Reply:
If I could make a suggestion on next steps. I think adding eslint (unless you are particularly attached to the style) and some tools like standard-version etc would help with contributions. Then it would be a matter of triage and prioritisation of bugs and features.